### PR TITLE
Removes a variable that is only set once

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Removes a practically unused variable

Rewrites some comments in `atoms.dm` because one was just wrong, and others were formatted weirdly or just plainly incorrect
(Looking at you, speech bubble)
image

## Why It's Good For The Game
Less variables on atoms mean less variables on a lot of things

## Testing
It compiled

## Changelog
NPFC

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -16,27 +16,29 @@
...
-	var/germ_level = GERM_LEVEL_AMBIENT // The higher the germ level, the more germ on the atom.
-	var/simulated = TRUE //filter for actions - used by lighting overlays
+	/// The higher the germ level, the more germ on the atom.
+	var/germ_level = GERM_LEVEL_AMBIENT
+	/// Filter for actions - used by lighting overlays
+	var/simulated = TRUE
 	var/atom_say_verb = "says"
-	var/bubble_icon = "default" ///what icon the mob uses for speechbubbles
-	var/dont_save = FALSE // For atoms that are temporary by necessity - like lighting overlays
+	///what icon the mob uses for speechbubbles
```
Comment (Oyu07) [inline code/game/atoms.dm:24]: Capital + Space

File: code/game/atoms.dm [hunk 2]
```diff
@@ -16,27 +16,29 @@
...
 	var/datum/reagents/reagents = null
 
-	//This atom's HUD (med/sec, etc) images. Associative list.
+	// This atom's HUD (med/sec, etc) images. Associative list.
 	var/list/image/hud_list
-	//HUD images that this atom can provide.
+	// HUD images that this atom can provide.
 	var/list/hud_possible
 
 	//Value used to increment ex_act() if reactionary_explosions is on
```
Comment (Oyu07) [inline code/game/atoms.dm:36]: Space

Comment (Oyu07) [inline code/game/atoms.dm:36]: Space

**Review by Oyu07 — COMMENTED** [review]
2 comment request since you touched them

File: code/game/atoms.dm (hunk 1 above)
Comment (DGamerL) [reply code/game/atoms.dm]: Did both!

**Review by Oyu07 — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**